dali/operators/generic/slice/slice.cc

```diff
@@ -46,6 +46,12 @@ When using positional inputs, two extra boolean arguments ``normalized_anchor``/
 can be used to specify the nature of the arguments provided. Using positional inputs for anchor
 and shape is incompatible with the named arguments specified above.
 
+For GPU backend and positional inputs ``anchor`` and ``shape``, both CPU and GPU data nodes
```

Comment:
Maybe we can make it a note. Just a thought.

Reply:
Done